### PR TITLE
Fix go test never fails

### PR DESCRIPTION
**Pull request by hugjobk**
@cnkarz go tests in kafkautils package were never run because you didn't write the TestMain properly.
You need to add this line at the end of TestMain function: `os.Exit(m.Run())`

After I've fixed TestMain function, go tests in kafkautils package failed. Can you fix them by adding new commit to this PR?

### CODE REVIEW

**Comment by hugjobk** [comment]
@cnkarz go tests in kafkautils package were never run because you didn't write the TestMain properly.
You need to add this line at the end of TestMain function: `os.Exit(m.Run())`

After I've fixed TestMain function, go tests in kafkautils package failed. Can you fix them by adding new commit to this PR?

**Comment by cnkarz** [comment]
why didn't you fix the tests?

**Comment by hugjobk** [comment]
> why didn't you fix the tests?

I could fix them but I thought it would be better to let you fix them since you are the author.

File: kafkautils/stream.go [hunk 1]
```diff
@@ -38,8 +38,10 @@ func NewSchema(schema string) (*StreamName, error) {
 		Event:     contractEvent[1],
 	}
 
-	aggregation := strings.SplitAfterN(contractEvent[1], TopicAggregateSeparator, 2)
+	aggregation := strings.Split(contractEvent[1], TopicAggregateSeparator)
```
Comment (rukolahasser) [inline kafkautils/stream.go:41]: They don't do the same thing. cuz `SplitAfterN` keeps the separator while `Split` does not.

Comment (hugjobk) [reply kafkautils/stream.go:41]: I think there was a bug in NewSchema function and @cnkarz changed from SplitAfterN to Split to fix it

**Review by rukolahasser — APPROVED** [review]

**Pull request merged**